report_py3o/__openerp__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2013 XCG Consulting (http://odoo.consulting)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'LibreOffice Report Engine',
```

Comment:
Same note regarding the application specific name. This is converting into the Open Document Format, as there is no such thing as LibreOffice format. These same templates should also work just fine in Microsoft Office.